packages/resolve-auth/src/helpers.js

```diff
@@ -0,0 +1,44 @@
+import cookie from 'cookie'
+
+const createRequest = expressReq => ({
+  path: expressReq.path,
+  method: expressReq.method,
+  originalUrl: expressReq.originalUrl,
+  headers: expressReq.headers,
+  params: expressReq.params,
+  body: expressReq.body,
+  query: expressReq.query,
+  cookies: expressReq.cookies,
+  resolve: expressReq.resolve
+})
+
+const createResponse = expressRes => {
+  const response = {
+    expressRes,
+    clearCookie: name => {
+      const fieldContentRegExp = /^[\u0009\u0020-\u007e\u0080-\u00ff]+$/ // eslint-disable-line
```

Comment:
Is it expected behavior that RegExp checks specified *codepoints* instead of *octets* ? (See www.utf8-chartable.de/unicode-utf8-table.pl?start=0&number=1024) ES engine works with UTF-16 set internally, and suppose Unicode codepoints in regexp with `\uXXXX` literals. It does not match potential underlying bytes in appropriate HTTP header, and also does not affect MIME encoding.